ex02.ex

```diff
@@ -41,7 +41,7 @@ defmodule Ex02 do
   # Do the same using the & syntax #
   ##################################
 
-  list2b = your_anonymous_function(1, 2)
+  list2b = &([ &1 + &2, &1 - &2 ])
```

Comment:
No deduction, but the unnecessary parens here make for a more confusing line. You could just have

~~~ elixir
list2b = &[ &1 + &2, &1 - &2 ]
~~~